03fibonacciNumbers.cpp

```diff
@@ -0,0 +1,52 @@
+#include <iostream>
+
+using namespace std;
+
+void fibonacciNumberPositionInput(int* fibonacciNumberPosition);

```

Comment:
`fibonacciNumberPositionInput` => `inputNumberPosition`
Functions start with a verb. Remember it.

Reply:
![image](https://user-images.githubusercontent.com/79978383/111069898-4fb90b80-84e0-11eb-8354-3c94332f0ee9.png)
Thank you!
done